packages/govuk-frontend/src/govuk/errors/index.mjs

```diff
@@ -0,0 +1,33 @@
+/**
+ * A base class for `Error`s thrown by GOV.UK Frontend.
+ *
+ * It is meant to be extended into specific types of errors
+ * to be thrown by our code.
+ *
+ * @example
+ * ```js
```

Comment:
Just a reminder to use `mjs` over `js` for ES modules code blocks

Can't remember if ESLint is set up for these (like it is for Markdown) but it might matter one day

Reply:
The example doesn't use anything module specific (`import`/`export`) so plain `js` is fine there 😊 

Regarding ESLint, it doesn't seem we've enabled [jsdoc/check-examples](https://github.com/gajus/eslint-plugin-jsdoc/blob/HEAD/docs/rules/check-examples.md#readme). We can discuss its enabling in a separate part as it may have some tricky side effects (eg. requiring this class to have its own JSDoc block inside the example).

Reply:
Yeah not so much the module specific stuff

We tend to use [`*.mjs` as the default now](https://github.com/alphagov/govuk-frontend/commit/4f6a747ce3b25e7a5323b772fd4ca47a8049780b), so some rules like the TypeScript parser are only for `*.mjs`

https://github.com/alphagov/govuk-frontend/blob/e1ef6d2df52908e1b87a8c692bc894adf015f57c/packages/govuk-frontend/.eslintrc.js#L11-L13